content/guides/destructuring.adoc

```diff
@@ -77,6 +79,15 @@ This type of destructuring can be used on any kind of data structure that can be
 
 The key to sequential destructuring is that you bind the values one-by-one to the symbols in the vector. For instance the vector `[x y z]` will match each element one-by-one with the list `'(1 2 3)`.
 
+If the same symbol appears several times in the left-hand destructuring form, the last binding for that symbol is retained:
```

Comment:
This is undefined and unimportant, should remove.

Reply:
Actually I was thinking about what happens when there are several "_", got confused (because this is not Prolog) then it occurred to me that this is just a symbol like any other and as for other symbol, later bindings will scratch earlier ones, hence this remark. 

Do you want me remove and resubmit a pull request?

Reply:
Yes